src/components/pages/metadata-view-page/components/MDVGridView.js

```diff
@@ -0,0 +1,121 @@
+// @flow
```

Comment:
Similarly, could we do without the `pages/metadata-view-page/components` path? Maybe put it at `src/components/grid-view`?